.github/workflows/build.yml

```diff
@@ -23,7 +23,6 @@ jobs:
         uses: actions-rs/cargo@v1
         with:
           command: check
-          
```

Comment:
Sometimes we leave newline between different actions sometimes not. Let's make it everywhere (newline between each action) please :)